.golangci.yml

```diff
@@ -148,6 +141,15 @@ linters-settings:
     range-loops: true # Report preallocation suggestions on range loops, true by default
     for-loops: false # Report preallocation suggestions on for loops, false by default
 
+issues:
+  # Excluding configuration per-path, per-linter, per-text and per-source
+  exclude-rules:
+    - linters: [errcheck]
```

Comment:
I feel like skipping the below rules are making sense. Please let me know if you have different idea.

Reply:
i'm curious, why ignore the first two?

Reply:
Oh I just removed the first rule, second one is to avoid the below
```
integration/main.go:26:42         staticcheck  SA1016: os.Kill cannot be trapped (did you mean syscall.SIGTERM?)
```

However, as syscall.SIGTERM is already trapped, I can remove the second rule as well. Please let me know if you have any other concern.